aggregator/src/main/java/com/redhat/cloud/notifications/DailyEmailAggregationJob.java

```diff
@@ -137,37 +151,29 @@ List<AggregationCommand> processAggregateEmailsWithOrgPref(LocalDateTime endTime
         return pendingAggregationCommands;
     }
 
-    @Deprecated(forRemoval = true)
-    List<AggregationCommand> processAggregateEmails(LocalDateTime endTime, CollectorRegistry registry) {
-
-        final CronJobRun lastCronJobRun = emailAggregationResources.getLastCronJobRun();
-        LocalDateTime startTime = lastCronJobRun.getLastRun();
-
-        Log.infof("Collecting email aggregation for period (%s, %s) and type %s", startTime, endTime, DAILY);
-
-        final List<AggregationCommand> pendingAggregationCommands =
-                emailAggregationResources.getApplicationsWithPendingAggregation(startTime, endTime)
-                        .stream()
-                        .map(aggregationKey -> new AggregationCommand(aggregationKey, startTime, endTime, DAILY))
-                        .collect(Collectors.toList());
-
-        Log.infof(
-                "Finished collecting email aggregations for period (%s, %s) and type %s after %d seconds. %d (orgIds, applications) pairs were processed",
-                startTime,
-                endTime,
-                DAILY,
-                SECONDS.between(endTime, LocalDateTime.now(UTC)),
-                pendingAggregationCommands.size()
-        );
-
-        pairsProcessed = Gauge
-                .build()
-                .name("aggregator_job_orgid_application_pairs_processed")
-                .help("Number of orgId and application pairs processed.")
-                .register(registry);
-        pairsProcessed.set(pendingAggregationCommands.size());
-
-        return pendingAggregationCommands;
+    private void sendIt(AggregationCommand aggregationCommand) {
+
+        Payload.PayloadBuilder payloadBuilder = new Payload.PayloadBuilder();
+        Map<String, Object> payload = JsonObject.mapFrom(aggregationCommand).getMap();
+        payload.forEach(payloadBuilder::withAdditionalProperty);
+
+        Action action = new Action.ActionBuilder()
+            .withBundle(BUNDLE_NAME)
+            .withApplication(APP_NAME)
+            .withEventType(String.format("%s-%s-%s", EVENT_TYPE_NAME, aggregationCommand.getAggregationKey().getBundle(), aggregationCommand.getAggregationKey().getApplication()))
```

Comment:
cc @josejulio @MikelAlejoBR

Reply:
It's still unclear to me why we can't only write to the EventLog table that has everything decoupled (i.e. no FKs) and use only the displayName of the EventLog's event-type :thinking:

Reply:
We could alter `Event#eventTypeDisplayName` value and put the additional information without adding the `details` field I mentioned, if that's what your suggesting. That would require event less effort in the backend and zero change in the frontend, so I like this even more than using `details` :smile:

Reply:
If I understand properly you are suggesting:
1. Adding only one application + event_type on bundle console to be able to receive aggregation events
2. Adding on propery on `notification_history#details` to be able to override the `eventTypeDisplayName`
3. Patching the event api to be able to return the custom `eventTypeDisplayName` in case of aggregation event

Form my point of view, I've to think and find a solution regarding following impacts: 
1.   The main aggregation event_type will be considered as regular event type, that involve the capacity to subscribe, unsubscribe to it, linking some templates etc...
2. Event log filter on event name have to be updated to look on regular event display name, but also on 'patched' display name, which can have an impact on api response time

@josejulio @gwenneg Could you please confirm my understanding ?